File: app/src/main/java/com/example/thiennguyen/api/bangtin/BinhLuan.java
package com.example.thiennguyen.api.bangtin;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class BinhLuan {

    @SerializedName("id")
    public int id;

    @SerializedName("idBaiViet")
    public int idBaiViet;

    @SerializedName("idNguoiBinhLuan")
    public int idNguoiBinhLuan;

    @SerializedName("noiDung")
    public String noiDung;

    @SerializedName("ngayBinhLuan")
    public Date ngayBinhLuan;
}

File: backendc#/Data/BinhLuan.cs
// Trong file /Data/BinhLuan.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

public class BinhLuan
{
    [Key]
    public int Id { get; set; }

    public int IdBaiViet { get; set; }

    public int IdNguoiBinhLuan { get; set; }

    public string NoiDung { get; set; }

    public DateTime NgayBinhLuan { get; set; }
}